appveyor.yml

```diff
@@ -20,7 +20,6 @@ test:
 
 skip_commits:
   files:
-    - '**/*.md'
```

Comment:
_dev_ is a protected branch and we can only check in if a pull request has been created and a successful AppVeyor build run. However, if the only change is to an .md file then an AppVeyor build won't be triggered so the checkin will be blocked. So, we need to run a build even if only .md files have changed.